src/Apia/Defaults.hs

```diff
@@ -97,7 +97,7 @@ setCheck opts config = setDumpTypes newOpts config
   where
     newOpts ∷ Options
     newOpts = case lookConfig "check" config of
-      Just  val → opts { optCheck = val}
+      Just  val → opts { optCheck = val }
```

Comment:
The previous space was right. I was talking about the spaces between `Just` and `val`.